Nodejs/Product/Nodejs/Project/DependencyNode.cs

```diff
@@ -28,54 +28,25 @@
 using VsCommands2K = Microsoft.VisualStudio.VSConstants.VSStd2KCmdID;
 
 namespace Microsoft.NodejsTools.Project {
-    internal class DependencyNode : HierarchyNode {
+    internal interface IDependencyNodeBasePath {
+        string GetRelativeUrlFragment();
+    }
```

Comment:
I can changes in this file. The intention is that `DependencyNode` does not actually depend on having a `DependencyNode` as it's parent, but rather some object that can provide a base path. We can mock out the latter object much more easily than mocking out a `DependencyNode`. But I never ended up going down the path of testing this code so it's not required.